crates/wasmparser/Cargo.toml

```diff
@@ -11,21 +11,34 @@ A simple event-driven library for parsing WebAssembly binary files.
 """
 edition = "2018"
 
+[dependencies]
+no-std-compat = { version = "0.4.1", default-features = false, features = ["compat_hash", "compat_sync", "alloc"] }
```

Comment:
Until we are sure that the `hashbrown` usage is secure we should use the `BTreeMap` and `BTreeSet`. My guess would be that there is no way to have it secure (even with SipHash) without `std` for randomness.

Reply:
I agree!